lib/controllers/root-controller.js

```diff
@@ -43,7 +43,7 @@ export default class RootController extends React.Component {
   static propTypes = {
     workspace: PropTypes.object.isRequired,
     commandRegistry: PropTypes.object.isRequired,
-    notificationManager: PropTypes.object.isRequired,
+    notificationManager: PropTypes.object.isReuired,
```

Comment:
:point_up: It looks like this one was spelled right the first time? Caught a stray backspace keypress maybe?